modules/fscloud/README.md

```diff
@@ -0,0 +1,96 @@
+# Financial Services Cloud Profile
+
+This is a profile for Event Notifications that meets Financial Services Cloud requirements.
+It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals.
+
+### Usage
+
+```hcl
+module "event_notification" {
+  source                    = "terraform-ibm-modules/event-notifications/ibm/modules/fscloud"
```

Comment:
this is not the correct source. Double slashes are required when referencing sub folders: `terraform-ibm-modules/event-notifications//ibm/modules/fscloud`

Reply:
Done.